DevDockerfile

```diff
@@ -1,21 +1,33 @@
-FROM ruby:2.4-slim-stretch
-RUN apt-get update -qq && apt-get install -y build-essential libpq-dev postgresql-client \
-    && rm -rf /var/lib/apt/lists/*
-
-ENV RAILS_ROOT /var/www/service_approval
-
-RUN mkdir -p $RAILS_ROOT/tmp/pids
-
-WORKDIR $RAILS_ROOT
-
-COPY Gemfile Gemfile
-
-RUN gem install bundler
-
-RUN bundle install
-
+FROM manageiq/ruby:latest
```

Comment:
This looks almost exactly like `Dockerfile` in this repo. If the image is going to be nearly identical, why have two?